File: src/codes/forced-compatibility/py/uncode.py
# レガシーAPI（旧仕様）
class LegacyPrinter:
    def print_text(self, text: str):
        print(f"*** {text} ***")

# 新しいコード（無理やり接続）
class ReportGenerator:
    def print_summary(self, text: str):
        printer = LegacyPrinter()  # ← 直接依存
        legacy_format = text.upper()  # ← 無理やり仕様に合わせる
        printer.print_text(legacy_format)  # ← 古いインターフェースにベタ依存

# 利用例
report = ReportGenerator()
report.print_summary("月次レポート")

File: src/codes/forced-compatibility/php/uncode.php
<?php

// レガシーAPI（旧仕様）
class LegacyPrinter {
  public function printText(string $text): void {
    echo "*** {$text} ***\n";
  }
}

// 新しいコード（無理やり接続）
class ReportGenerator {
  public function printSummary(string $text): void {
    $printer = new LegacyPrinter(); // ← 古いクラスを直接使っている
    $legacyFormat = strtoupper($text); // ← 無理やり仕様に合わせる
    $printer->printText($legacyFormat); // ← ハードコーディング
  }
}

// 利用例
$report = new ReportGenerator();
$report->printSummary("月次レポート");
